packages/cli/src/templates/package.json

```diff
@@ -5,19 +5,22 @@
   "private": true,
   "type": "module",
   "scripts": {
-    "build": "yarn codegen && yarn compile",
+    "build": "yarn codegen && yarn lint --fix && yarn compile",
```

Comment:
I wouldn't add this to build, if it fails it is not possible to compile

Reply:
I get your point, but the thing is that we are running codegen before compiling, and that means that the autogenerated code does not comply with the lint rules. As a user, I would not be super comfortable if I had to manually run lint after each build because the file format would be broken.

How could we address this?

Reply:
How could we not take into account those files if the user can currently choose the output path with the --output flag of codegen?

Reply:
Solved: removed lint from build command and ignored default types path from eslint config